LumpNav/LumpNav.py

```diff
@@ -40,18 +40,71 @@ def __init__(self, parent = None):
     GuideletWidget.__init__(self, parent)
     
   def setup(self):
+    # Adds default configurations to Slicer.ini
+    self.addDefaultConfiguration()
+    
     GuideletWidget.setup(self)
-
+    
   def addLauncherWidgets(self):
     GuideletWidget.addLauncherWidgets(self)
 
+    # Configurations
+    self.configurations()
+    
+    # BreachWarning
+    self.breachWarningLight()  
+ 
+  # Adds default configurations to Slicer.ini
+  def addDefaultConfiguration(self):
+    settings = slicer.app.userSettings()    
+    if not settings.value(self.moduleName + '/Configurations/default/TipToSurfaceDistanceCrossHair'):

```

Comment:
Use contains (http://doc.qt.io/qt-4.8/qsettings.html#contains) or something similar to check for the existence of configurations/deafult instead of checking the existence of a particular value.